platforms/database.go

```diff
@@ -5,6 +5,16 @@ import (
 	"strings"
 )
 
+// isKnownOS returns true if the operating system is Linux.
+//
+// The OS value should be normalized before calling this function.
+func isLinuxOS(os string) bool {
```

Comment:
Why are these functions (isLinuxOS, isArmArch) in `database.go` instead of `cpuinfo.go` where they are used?

Reply:
Hi Daniel,
These functions are use by cpuinfo.go and cpuinfo_test.go.
As Stephen's comment on first version, functions like this belong in database.go.